src/lib.rs

```diff
@@ -25,7 +26,8 @@ pub struct MetricProvider {
 
 impl MetricProvider {
     pub fn new(config: Config) -> Self {
-        let register = Registry::new_custom(None, None).expect("Unable to build Registry");
+        let register = Registry::new_custom(Some("monitorbot".to_string()), None)
```

Comment:
We also need to update the rate limit collector right now, otherwise its metrics will be called `monitorbot_monitorbot_github_rate_limit`.

Reply:
fixed